solidity/hardhat.config.ts

```diff
@@ -31,6 +31,8 @@ const config: HardhatUserConfig = {
 
   networks: {
     hardhat: {
+      allowUnlimitedContractSize: true,
+      blockGasLimit: 30000000,
```

Comment:
Wow, 30M is large. Do we need to set it up that high? 

https://ethereum.org/en/developers/docs/gas/#block-size

Reply:
The `blockGasLimit` is no longer needed. I removed it in b0eb1f3c825cee422c76c8b0d5fdccdf88397f7a.